tests/phpunit/tests/query.php

```diff
@@ -769,4 +769,21 @@ static function() {
 		$this->assertArrayHasKey( 'join', $posts_clauses_request );
 		$this->assertSame( '/* posts_join_request */', $posts_clauses_request['join'] );
 	}
+
+	/**
+	 * Test that is_post_type_archive() returns false for an undefined post type.
+	 *
+	 * @ticket 56287
+	 */
+	public function test_is_post_type_archive_handles_null_post_type() {
```

Comment:
```suggestion
	public function test_is_post_type_archive_should_return_false_for_an_undefined_post_type() {
```
Describing _how_ `is_post_type_archive()` should handle this case in the test name makes it easier to see what's being tested. Replacing `null_post_type` with `an_undefined_post_type` is clearer and also matches the test docblock.